packages/react-native/src/index.ts

```diff
@@ -12,10 +12,9 @@ import { SDKBaseReactNative } from './react-native-sdk-base';
 
 // We expect `global.process` to be a Node Process,
 // so we replace it here.
-/* istanbul ignore next */
-if (typeof global.process === 'undefined') global.process = processPolyfill;
+global.process = processPolyfill || {};
```

Comment:
I'm wondering if this might cause issues for developers that mutate `global.process` themselves...

Reply:
They might lose their ENV variables, for instance.

Reply:
I agree with your concern. The issue I was trying to resolve is that there is a case where `process` has only `ENV` property in the entire object, while web3.js was trying to call `process.version`. This would make it impossible to use.

 Two alternatives:
* To have the process set as a developer's manual configuration only when they use web3.js
* To have this set up in a different package.

Reply:
Maybe a different package as `@magic-sdk/react-native-web3-polyfill`... or, would it work to use an object spread? Like:

```
if (global.process) global.process = { ...global.process, ...require('process') };
else global.process = require('process');
```

Reply:
Maybe we could also open an issue with Web3 and simply recommend Ethers.js for React Native use-cases officially for now.

Reply:
I think object spread is good enough. I have also checked another guide https://medium.com/@talaikis/react-native-0-60-4-with-web3-js-1-2-0-8090a1decee2, it does object spread as well. 

I'll do object-spread.